.github/actions/load-workflow-variables/action.yml

```diff
@@ -33,11 +33,20 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: 'Get Approved Configuration File'
+      uses: 'actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11' # ratchet:actions/checkout@v4
+      with:
+        ref: '${{ github.event.repository.default_branch }}'
+        path: '${{ github.sha }}'
```

Comment:
Should we prefix this with the action name to prevent collisions? Or use a tmp dir?

Reply:
opted for the former. The Checkout action restricts the destination path to within the current working directory. Also cannot change the `working_directory` when using `workflow_call` 👎